### PR TITLE
Add get all food items route

### PR DESCRIPTION
**Pull request by Link-**
This PR creates a new route with the following details:

```
Verb: GET
Route: /all-food-items
Response: JSON

Response-example:
{"item1":"Banana","item2":"Apple","item3":"Strawberry"}
```

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -19,4 +19,12 @@
 
 Route::get('/new_message', function () {
     return 'New message!';
-});
```
Comment (mohamadwehbe) [inline routes/web.php:22]: deleted

File: routes/web.php [hunk 2]
```diff
@@ -19,4 +19,12 @@
...
\ No newline at end of file
+});
+
+Route::get('all-food-items', function () { 
+    return response()->json([
+        'item1'=>'Banana',
+        'item2'=>'Apple',
+        'item3'=>'Strawberry',
```
Comment (mohamadwehbe) [inline routes/web.php:24]: hayaa hi

**Review by mohamadwehbe — CHANGES_REQUESTED** [review]

File: routes/web.php (hunk 1 above)
Comment (Link-) [reply routes/web.php:22]: This was fixed

File: routes/web.php (hunk 2 above)
Comment (Link-) [reply routes/web.php:24]: Hi

**Pull request merged**